concepts/05 Widgets/Tabs/05 Customize Item Appearance.md

```diff
@@ -1,103 +1,326 @@
 For a minor customization of tabs, you can define [specific fields](/api-reference/10%20UI%20Widgets/dxTabs/1%20Configuration/items '/Documentation/ApiReference/UI_Widgets/dxTabs/Configuration/items/') in item data objects. For example, the following code generates three tabs: the first has an icon, the second has a [badge](/api-reference/_hidden/dxTabPanelItem/badge.md '/Documentation/ApiReference/UI_Widgets/dxTabPanel/Configuration/items/#badge'), the third is disabled.
 
+---
+##### jQuery
+
+    <!--HTML-->
+    <div id="tabsContainer"></div>
+
     <!--JavaScript-->
     $(function() {
         $("#tabsContainer").dxTabs({
             items: [
                 { text: "User", icon: 'user' },
                 { text: "Comment", badge: "New" },
-                { text: "Find", disabled: true }
+                { text: "Find" }
```

Comment:
It looks like this example should be in the Overview article, and the example from there should be in this article.